test/sanity/code-smell/no-unwanted-files.py

```diff
@@ -36,6 +38,8 @@ def main():
             continue
 
         ext = os.path.splitext(path)[1]
+        if ext in ('.yml', '.yaml') and any(path.startswith(yaml_directory) for yaml_directory in allow_yaml):
```

Comment:
```suggestion
        if ext in ('.yml', ) and any(path.startswith(yaml_directory) for yaml_directory in allow_yaml):
```
This test is only run for the ansible-core repo, and since you only want to allow `.yml` files here, you shouldn't allow `.yaml` files..

Reply:
i thought it was shared with collections

Reply:
The shared code-smell tests are in test/lib/ansible_test/_util/controller/sanity/code-smell/.

Reply:
so the shared code is in the 'controller' specific dir ... not the 'non specific' dir ... TIL ...